recipes/pyaez/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "PyAEZ" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 061ccd6f2a4d2496b84fa5b65c0134b1e0935e68b3ceab78aac4bc9d8ed7d0f2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pip 
+    - numpy
+    - numba
+    - scipy
+    - gdal
+
+test:
+  imports:
+    - pyaez
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/gicait/pyaez
+  summary: 'PyAEZ is a python package consisted of many algorithms related to Agro-ecalogical zoning (AEZ) framework. PyAEZ tries to encapsulate all complex calculations in AEZ and try to provide user friendly, and intuitive ways to input data and output results after calculations.'
```

Comment:
The summary is as big, actually the same, as the description. Upstream has it as:

```
PyAEZ is a python package consisted of many algorithms related to Agro-ecalogical zoning (AEZ) framework.
```

Reply:
I just make it shorter.